attributes/default.rb

```diff
@@ -81,13 +82,13 @@
 default['hopslog']['private_ips']         = ['10.0.2.15']
 default['hopslog']['public_ips']          = ['10.0.2.15']
 
-# Kibana Opendistro Security plugin
-default['kibana']['opendistro_security']['url']                                   = "#{node['download_url']}/opendistro_security_kibana_plugin-#{node['elastic']['opendistro']['version']}.zip"
-default['kibana']['opendistro_security']['https']['enabled']                      = "true"
-default['kibana']['opendistro_security']['multitenancy']['global']['enabled']     = "false"
-default['kibana']['opendistro_security']['multitenancy']['private']['enabled']    = "true"
-default['kibana']['opendistro_security']['cookie']['ttl']                         = node['elastic']['opendistro_security']['jwt']['exp_ms'].to_i
+# Kibana Opensearch Security plugin
+default['kibana']['opensearch_security']['url']                                   = "#{node['download_url']}/opensearch_security_kibana_plugin-#{node['elastic']['opensearch']['version']}.zip"
+default['kibana']['opensearch_security']['https']['enabled']                      = "true"
+default['kibana']['opensearch_security']['multitenancy']['global']['enabled']     = "false"
+default['kibana']['opensearch_security']['multitenancy']['private']['enabled']    = "true"
+default['kibana']['opensearch_security']['cookie']['ttl']                         = node['elastic']['opensearch_security']['jwt']['exp_ms'].to_i
 # the session ttl time is set to be twice the time the cookie ttl time, in order to solve the session expiry issue in kibana_addr
-# https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/issues/31
-default['kibana']['opendistro_security']['session']['ttl']                        = 2 * node['elastic']['opendistro_security']['jwt']['exp_ms'].to_i
-default['kibana']['opendistro_security']['session']['keepalive']                  = "true"
+# https://github.com/opensearch-for-elasticsearch/security-kibana-plugin/issues/31
```

Comment:
Link doesn't work anymore, i wonder if this is an issue anymore though, will need to test it
https://github.com/opensearch-project/security-dashboards-plugin/pull/560
```suggestion
# https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/issues/31
```